widgets/src/map/ChoroplethCountries.js

```diff
@@ -66,9 +86,9 @@
         this.choroPaths = choroPaths.enter().append("path")
             .attr("d", this.d3Path)
             .on("click", function (d) {
-                var code = countries.countryNames[d.id];
-                if (context._dataMap[code]) {
-                    context.click(context.rowToObj(context._dataMap[code]), "weight");
+                if (context._dataMap[d.id]) {
+                    var obj = [d.id, d.name, context._dataMap[d.id]];
```

Comment:
The click information is wrong: {"Country":156,"Weight":"China","Label":229.946185501741}, weight